entries/l/line6.com.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Line 6": {
+    "domain": "line6.com",
+    "tfa": [
+      "totp"

```

Comment:
I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
![screenshot](https://user-images.githubusercontent.com/3535780/187552643-08eea7e8-e6c6-4873-b31c-0faf4347f69f.gif)

Reply:
Thanks for the clarification! Unfortunately I have not had luck finding public docs but I can definitely add some screen shots.

Instructions:
1. [Log in](https://line6.com/account)
2. Go to [Two Factor Authentication](line6.com/account/2fa/) on the left menu.
3. Scan the QR code with your chosen 2FA app or enter the provided key.
![Screenshot 2023-02-05 234540](https://user-images.githubusercontent.com/2245265/216886221-77642cd0-cd2f-4cd2-ba25-dfc7c171a06d.png)

Reply:
Thanks!